bluesky/suspenders.py

```diff
@@ -533,7 +533,8 @@ class SuspendWhenChanged(SuspenderBase):
     
     Only resume if allowed AND when monitored equals expected.
     
-    .. note: this suspender is designed to require bluesky restart if value changes
+    .. note:
```

Comment:
I think this needs a blank line below `.. note` to render correctly in HTML.

Reply:
Of course.